ext/storage_sources/azure_store/azure_storage_source.cpp

```diff
@@ -662,6 +677,20 @@ azure_file_size(WT_FILE_HANDLE *file_handle, WT_SESSION *session, wt_off_t *size
     return 0;
 }
 
+// Log collected statistics.
+static void
+azure_log_statistics(const azure_store &azure_storage)
+{
+    azure_storage.log->log_debug_message(
+      "GCP list objects count: " + std::to_string(azure_storage.statistics.list_objects_count));
```

Comment:
These are using still using GCP naming

Reply:
Oops! thank you for catching it out [fcb039f](https://github.com/wiredtiger/wiredtiger/pull/8932/commits/fcb039f1e2c4c0dbf1e68bb8dad0aff3bebc9f24)